torch/csrc/autograd/python_function.h

```diff
@@ -27,6 +27,11 @@ struct PyNode : public Node {
   variable_list apply(variable_list&& inputs) override;
   variable_list legacy_apply(const variable_list& inputs);
 
+  // throw a persist python_error to allow it to be thrown
```

Comment:
Maybe tweak the comment? 
 // Throw a python_error with the PyErr state persisted,
 // so that we don't lose the error state if the GIL is released,
 // or if the exception is passed to another thread.